File: 2/PHP/Hardware.php
<?php

class Hardware extends Product{
	// Atribut Private
	private $brand;
	private $model;

	// Konstruktor
	function __construct() {
		$brand = "";
		$model = "";
	}

	// Set & Get

	// Brand
	function setBrand($brand) {
		$this->brand = $brand;
	}
	function getBrand() {
		return $this->brand;
	}

	// Model
	function setModel($model) {
		$this->model = $model;
	}
	function getModel() {
		return $this->model;
	}
}

?>

File: 2/C++/Hardware.cpp
class Hardware: public Product {
	private:
		// Atribut
		string brand;	// Brand
		string model;	// Model

	public:
		// Konstruktor
		Hardware() {
			brand = "";
			model = "";
		}

		/* Set & Get */

		// Brand
		void setBrand(string brand) {
			this->brand = brand;
		}
		string getBrand() {
			return brand;
		}

		// Model
		void setModel(string model) {
			this->model = model;
		}
		string getModel() {
			return model;
		}

		// Destruktor
		~Hardware(){}
};